3d_armor/README.md

```diff
@@ -1,27 +1,28 @@
-|- - [armor_punch_damage](#enable-punch-damage-effects)                      |||- - - [armor:remove_all](#armor-remove_all)
-|- - [armor_migrate_old_inventory](#migration-of-old-armor-inventories)      |||- - - [armor:equip](#armor-equip)
-|- - [wieldview_update_time](#how-often-player-wield-items-are-updated)      |||- - - [armor:unequip](#armor-unequip)
-| - [Credits](#credits)                                                      |||- - - [armor:update_skin](#armor-update_skin)
-|                                                                            |||- - [Callbacks](#Callbacks)
-|                                                                            |||- - - [Item callbacks](#item-callbacks)
-|                                                                            |||- - - [Global callbacks](#global-callbacks)
+|-[Overview](#overview)														|||-[API](#api)
+|-[Armor Configuration](#armor-configuration)								|||- - [3d_Armor Item Storage](#3d_armor-item-storage)
+|- - [disable_specific_materials](#to-disable-individual-armor-materials)	|||- - [Armor Registration](#armor-registration)
+|- - [armor_init_delay](#initialization-glitches-when-a-player-first-joins)	|||- - [Registering Armor Groups](#registering-armor-groups)
+|- - [armor_init_times](#number-of-initialization-attempts)					|||- - [Groups used by 3d_Armor](#groups-used-by-3d_armor)
+|- - [armor_bones_delay](#armor-not-in-bones-due-to-server-lag)				|||- - - [Elements](#elements)
+|- - [armor_update_time](#how-often-player-armor-items-are-updated)			|||- - - [Attributes](#attributes)
+|- - [armor_drop](#drop-armor-when-a-player-dies)							|||- - - [Physics](#physics)
+|- - [armor_destroy](#destroy-armor-when-a-player-dies)						|||- - - [Durability](#durability)
+|- - [armor_level_multiplier](#armor-level-multiplyer)						|||- - - [Armor Material](#armor-material)
+|- - [armor_heal_multiplier](#armor-healing-multiplyer)						|||- - [Armour Functions](#armor-functions)
+|- - [armor_set_elements](#allows-the-customisation-of-armor-set)			|||- - - [armor:set_player_armor](#armor-set_player_armor)
+|- - [armor_set_bonus](#armor-set-bonus-multiplier )						|||- - - [armor:punch](#armor-punch)
+|- - [armor_water_protect](#enable-water-protection)						|||- - - [armor:damage](#armor-damage)
+|- - [armor_fire_protect](#enable-fire-protection)							|||- - - [armor:remove_all](#armor-remove_all)
+|- - [armor_punch_damage](#enable-punch-damage-effects)						|||- - - [armor:equip](#armor-equip)
+|- - [armor_migrate_old_inventory](#migration-of-old-armor-inventories)		|||- - - [armor:unequip](#armor-unequip)
+|- - [wieldview_update_time](#how-often-player-wield-items-are-updated)		|||- - - [armor:update_skin](#armor-update_skin)
+|-[Credits](#credits)														|||- - [Callbacks](#Callbacks)
+|																			|||- - - [Item callbacks](#item-callbacks)
+|																			|||- - - [Global callbacks](#global-callbacks)
```

Comment:
Please use spaces to align. This is hurts to read.

Reply:
Drives me nuts to, Ill fix that up looks like when I rebuilt it using concat in excel its added in some funky spacing.